api/slash-z-inner.js

```diff
@@ -0,0 +1,242 @@
+import Prisma from './prisma.js'
+import isPublicSlackChannel from './is-public-slack-channel.js'
+import userIsRestricted from './user-is-restricted.js'
+import channelIsForbidden from './channel-is-forbidden.js'
+import openZoomMeeting from './open-zoom-meeting.js'
+import transcript from './transcript.js'
+import fetch from 'node-fetch'
+
+export default async ({
+  displayName,
+  userId,
+  channelId,
+  responseUrl,
+  skipWarning = false
+}) => {
+  console.log({
```

Comment:
Could you remove the `console.log` if you don't need it? If it'd help for debugging in prod feel free to keep it in!

Reply:
This was in the original code so I kept it, but I can remove it if you'd like!

Reply:
Yes please!